examples/twri/data/gen_data_bg_acou.jl

```diff
@@ -0,0 +1,57 @@
+################################################################################
+#
+# Generate synthetic data for the BG Compass model
+#
+# Author: Mathias Loubout and Gabbrio Rizzuti
+# Date: September 2020
+################################################################################
+### Module loading
+
+using JUDI, JLD2, JUDI.TimeModeling
```

Comment:
Just a general remark—here the user has to install JLD2 herself? I understand that we do no want to add unnecessary packages to the dependencies but on the other hand here the user will encounter an error most of the time.

Reply:
Well  two things
- (some) people are already complaining that there is too many dependencies. 
- Adding JLD2  to the dependencies would not fi the issue since JLD2  would be available to JUDI (`julia --project`) but not to the user.

But should add remarks or something to tell what people need for the examples.